pkg/util/winutil/winctrlhandler.go

```diff
@@ -0,0 +1,45 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+//go:build windows
+
+// Package winutil contains Windows OS utilities
+package winutil
+
+import "syscall"
+
+var (
+	setConsoleCtrlHandler = k32.NewProc("SetConsoleCtrlHandler")
+)
+
+const (
+	// CtrlCEvent is code for Cntrl+C event
+	CtrlCEvent = 0
+	// CtrlBreakEvent is code for Cntrl+Break event
+	CtrlBreakEvent = 1
+)
```

Comment:
suggestion: linter should be okay with a comment on the whole block

```suggestion
// Console control signal constants
//
// https://learn.microsoft.com/en-us/windows/console/handlerroutine
const (
	CtrlCEvent = 0
	CtrlBreakEvent = 1
)
```